CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added 'point in time' APIs to the pyi files in sync and async client ([#378](https://github.com/opensearch-project/opensearch-py/pull/378))
 - Added MacOS and Windows CI workflows ([#390](https://github.com/opensearch-project/opensearch-py/pull/390))
 - Compatibility with OpenSearch 2.1.0 - 2.6.0 ([#381](https://github.com/opensearch-project/opensearch-py/pull/381))
+- Added a `collapse` method to `opensearchpy.helpers.search.Search` ([#408](https://github.com/opensearch-project/opensearch-py/issues/408))
```

Comment:
Hello @qcoumes, just a small correction. Please change PR No. to 409

Reply:
Done !